docs/advanced.md

```diff
@@ -162,3 +162,61 @@ client = httpx.Client(proxies=proxy)
 # This request will be tunneled instead of forwarded.
 client.get("http://example.com")
 ```
+
+
+## Timeout fine-tuning
+HTTPX library offers many request timeout management options. HTTPX distinguish three types of timeouts: **connect** timeouts, 
+**write** timeouts and **read** timeouts.
+
+* Connect timeout specify time period within which a connection between a client and a server must be established.
```

Comment:
Phrasing suggestion, and also an idea about documenting what happens in case of timeout:

```suggestion
* The **connect timeout** specifies the maximum amount of time to wait until a connection to the requested host is established. If HTTPX is unable to connect within this time frame, a `ConnectTimeout` exception is raised.
```